.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: Build Bluejay Firmware.
+
+on:
+  push:
+    branches:
+    - main
+    tags:
+    - '*'
+  pull_request:
+    branches:
+    - main
```

Comment:
Pull requests will be directed against develop branch in the future. Main branch should always stay on the latest release state.

Reply:
Yeah we will make both main and develop protected branches. Nothing gets merged without the github actions checks passing.

Right now checks passing = just build successful and some maintainer approves. But eventually it would be very nice to have automated tests running on real ESCs with arduino-dshot (via. ESP32 or something) or even a raspberry pi (If we can't get a public IP for ESP32 for eg., we can run github actions runner on raspberry pi iirc) and report the status back to github actions.